sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -287,8 +287,15 @@ def to_arrow(self) -> pyarrow.Table:
 
 
 def block_until_done(client, bq_job):
+    def _cancel_job(job_id):
```

Comment:
i don't think this needs to be in its own method. It's actually easier to see what's going on when these two lines are explicitly stated at the location where it's called.